api/server/server.go

```diff
@@ -1313,8 +1330,9 @@ func createRouter(eng *engine.Engine, logging, enableCors bool, dockerVersion st
 			"/exec/{name:.*}/resize":        postContainerExecResize,
 		},
 		"DELETE": {
-			"/containers/{name:.*}": deleteContainers,
-			"/images/{name:.*}":     deleteImages,
+			"/containers/{name:.*}/logs": deleteContainersLogs,
```

Comment:
Why is this a new API endpoint and not a query param on the existing logs endpoint?